tests/spec/026-shortcuts-notification.js

```diff
@@ -11,23 +10,6 @@ import { loginAsFoobar } from '../roles'
 fixture`026-shortcuts-notification.js`
   .page`http://localhost:4002`
 
-test('Shortcut f toggles favorite status in notification', async t => {
-  const idx = 0
-  await loginAsFoobar(t)
-  await t
-    .expect(getUrl()).eql('http://localhost:4002/')
-    .click(notificationsNavButton)
-    .expect(getUrl()).contains('/notifications')
-    .expect(getNthStatus(1 + idx).exists).ok({ timeout: 30000 })
-    .expect(getNthFavoritedLabel(1 + idx)).eql('Favorite')
-    .pressKey('j '.repeat(idx + 1))
-    .expect(isNthStatusActive(1 + idx)()).ok()
-    .pressKey('f')
-    .expect(getNthFavoritedLabel(1 + idx)).eql('Unfavorite')
-    .pressKey('f')
-    .expect(getNthFavoritedLabel(1 + idx)).eql('Favorite')
-})
```

Comment:
Well that's kind of a shame that the shortcuts don't work anymore. But I guess it isn't the end of the world.

Reply:
I've added this test back in: fda15ac8 . We can just favorite another toot.